packages/dd-trace/test/appsec/standalone.spec.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
+
+const { channel } = require('dc-polyfill')
+const startCh = channel('dd-trace:span:start')
+
+const standalone = require('../../src/appsec/standalone')
+const DatadogSpan = require('../../src/opentracing/span')
+const { APM_TRACING_ENABLED_KEY } = require('../../src/constants')
+
+describe('Appsec Standalone', () => {
+  let config
+  let tracer, processor, prioritySampler
+
+  beforeEach(() => {
+    config = { appsec: { standalone: { enabled: true } } }
+
+    tracer = {}
+    processor = {}
+    prioritySampler = {}
+  })
+
+  afterEach(sinon.restore)
```

Comment:
I think that is better if we are explicit with functions like `before()` `afterEach()`... These methods have different behaviour if they accept params or not... it is better if we don't link the behaviour with sinon methods.
```suggestion
  afterEach(() => { sinon.restore() })
```